src/main/java/com/dev/movietheatre/Main.java

```diff
@@ -50,5 +52,16 @@ public static void main(String[] args) throws AuthenticationException {
         User loginedUser = authenticationService.login("Billy@ukr.net", "111");
         System.out.println("Registered user " + registeredUser.toString());
         System.out.println("Logined user " + loginedUser.toString());
+
+        ShoppingCartService shoppingCartService = (ShoppingCartService) injector
+                .getInstance(ShoppingCartService.class);
+        User registeredUser1 = authenticationService.register("Sam@ukr.net", "000");
+        shoppingCartService.addSession(movieSession,registeredUser1);
+        ShoppingCart shoopingCartbyUser1 = shoppingCartService.getByUser(registeredUser1);
```

Comment:
```suggestion
        ShoppingCart shoppingCartByUser1 = shoppingCartService.getByUser(registeredUser1);
```
please be careful with spelling and camelCase

Reply:
Sorry, i will be more careful.